bin/improver-standardise

```diff
@@ -107,18 +100,31 @@ def main():
 
     parser.add_argument("--json_file", metavar="JSON_FILE", default=None,
                         help='Filename for the json file containing required '
-                             'changes to the metadata. Defaults to None.')
+                             'changes that will be applied '
+                             'to the metadata. Defaults to None.')
 
     args = parser.parse_args()
 
     # source file data path is a mandatory argument
     source_data = load_cube(args.source_data_filepath)
 
+    if args.fix_float64:
+        check_cube_not_float64(source_data, fix=True)
+    else:
+        check_cube_not_float64(source_data, fix=False)
+
+    if args.target_grid_filepath or args.json_file or args.fix_float64:
+        if not args.output_filepath:
+            msg = "An argument has been specified that requires an output " \
```

Comment:
It's slightly preferable to use brackets for line continuation, rather than a slash i.e.
```python
msg = ("An argument has been specified that requires an output "
       "filepath but none has been provided.")
```

Reply:
Fixed